### PR TITLE
feat(comp): Center component

### PR DESCRIPTION
**Pull request by ti10le**
#1860

### CODE REVIEW

**Comment by sebald** [comment]
@ti10le add gap? :)

File: docs/content/components/center.mdx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+## Properties (API)
+
+| Property         | Type                           | Default  |
+| :--------------- | :----------------------------- | :------- |
+| `maxHeight`      | `string`                       |          |
+| `maxWidth`       | `string`                       |          |
+| `textAlign`      | `left, center, right`          | `center` |
+| `space`          | `ResponsiveStyleValue<string>` | `none`   |
+| `superCenntered` | `boolean`                      | `false`  |
```
Comment (sebald) [inline docs/content/components/center.mdx:27]: Shouldn't the layout always be centered? "Max height" didn't make sense otherwise 🤔

**Comment by ti10le** [comment]
Currently we do not know where and in which cases we can use center. Therefore, we leave it out for now and add it when needed.

**Comment by ti10le** [comment]
@sebald I've made some rewrite. It's not for super centering with an position absolute.
orientation: [every-layout](https://every-layout.dev/layouts/center/) & [chakra](https://chakra-ui.com/docs/components/layout/center)

**Comment by sebald** [comment]
Will take a look at it tomorrow :)

**Review by sebald — APPROVED** [review]

**Pull request merged**